account/user_blackbox_test.go

```diff
@@ -95,7 +95,8 @@ func (s *userBlackBoxTest) TestExistsUser() {
 		// Check not existing
 		err := s.repo.CheckExists(s.ctx, uuid.NewV4().String())
 		// then
-		require.IsType(s.T(), &goa.ErrorResponse{}, err)
+		var errorType errors.NotFoundError
+		require.IsType(s.T(), errorType, err)
```

Comment:
Does it work if your instead write `require.IsType(s.T(), errors.NotFoundError{}, err)` ?